### ISSUE TITLE
Provide way to allow distributions to require openeuropa/oe_theme

### ISSUE THREAD

**Issue by ghost** [openeuropa/oe_theme#424]
When trying to build an openeuropa/distribution kind of package we found that using the artifact approach here can only be used in the root composer.json and not in the composer.json of a dependency of the root project:

https://github.com/openeuropa/drupal-site-template/blob/master/composer.json#L45-L52
vs
https://github.com/verbruggenalex/oesd/blob/master/composer.json#L58-L65

We should for sure improve the documentation here on how to install the openeuropa/oe_theme package:
https://github.com/openeuropa/oe_theme/blob/6180695f4374ab82afa888286878c629776da64b/README.md#installation

But if possible at all I would prefer that we alter the plugin here to also check the extra sections of other dependencies that require the plugin:  https://github.com/openeuropa/composer-artifacts/blob/master/src/Plugin.php#L51

**Comment by ghost** [comment]
On second thought, I would prefer to create two extra branches (e.g. 1.x-build, 2.x-build).
And instead of doing this: https://github.com/openeuropa/oe_theme/blob/2.x/.drone.yml#L76-L81

Commit the build theme each time it passes the tests to the new branch. And tag those branches so packagist.org will grab that instead of the source code.

What do you guys think about that approach? @ademarco, @imanoleguskiza , @sergepavle

**Comment by ademarco** [comment]
Yeah, I'd rather improve the documentation on the theme, composer artifacts is supposed to be configured on the project root.
